### DISCUSSION TITLE
Search in Documentation can be better.

### DISCUSSION THREAD

**Discussion by giray123** [vuetifyjs/vuetify#13589] [General]
Hi, I do not like the new search on the documentation site. I do not know exactly how long ago it has been changed in terms of UX or I do not know if this change is due to Vuetify or Angolia. However, I think it has made finding stuff harder:

1. I can not use keyboard shortcuts after the search. I want to press "/" to focus search field (this works), type a word, use arrow functions and press enter. It was like this before for a long time.
2. I think search results can be much better. When I type "text" for example, I would want text-field and textarea components to come as first results and I prefer component pages on the top and API pages as second. This is not the case for now. There are much irrelevant stuff on top that are rarely used. I believe this kind of stuff can be set with Angolia.

I think a better search would be really good for usage. What are your thoughts about this?

**Comment by Maxim-Mazurok** [comment]
Totally agree, same experience. I ended up not using this search at all.
When I want to find a component - I use browser search on page (Ctrl+F). Have to make sure that components panel is open first.
When I'm looking for relevant results, I use google with `site:http://vuetifyjs.com/ fill height`, for example.

You can try searching for `fill height`. It will suggest "file ... height" before the Grid system page which actually has info about `fill-height` helper. And even then it's shown truncated: "fill-" without "height".
